app/serializers/site_text_serializer.rb

```diff
@@ -11,6 +11,14 @@ def value
     object[:value]
   end
 
+  def interpolation_keys
+    object[:interpolation_keys]
+  end
+
+  def has_interpolation_keys?
+    !object[:interpolation_keys].empty?
```

Comment:
Wouldn't that work?

```suggestion
    object[:interpolation_keys].present?
```

Reply:
You're right. And it would save from a freak accident if it happens to be `nil` as well. 👍